server/.env.default

```
@@ -7,9 +7,9 @@ SERVICE_ACCOUNT_ENCRYPTION_KEY="ldRWi1hLL8zMT2yxAXwjjTJUYs1ikROLqRxFrs0qpvU=" #
 EMBEDDING_MODEL="bge-base-en-v1.5"
 
 # AWS Config for SES (different from Bedrock keys)
-AWS_ACCESS_KEY_ID=YourAWSAccessKeyID
-AWS_SECRET_ACCESS_KEY=YourAWSSecretAccessKey
-AWS_REGION=eu-north-1
+SES_AWS_ACCESS_KEY_ID=YourAWSAccessKeyID
+SES_AWS_SECRET_ACCESS_KEY=YourAWSSecretAccessKey
+SES_AWS_REGION=eu-north-1
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SES_AWS_REGION key should go before the SES_AWS_SECRET_ACCESS_KEY key

(UnorderedKey)